app/services/idv/agent.rb

```diff
@@ -18,6 +18,7 @@ def proof(*stages)
         log_vendor(vendor, results, stage)
         proofer_result = vendor.proof(@applicant)
         results = merge_results(results, proofer_result)
+        results[:timed_out] = true if proofer_result.timed_out?
```

Comment:
Style: another option would be `results[:timed_out] = proofer_result.timed_out?` (assuming `#timed_out?` always returns `true` or `false`).

Reply:
This actually doesn't work. We want this to be false if the first result times out, but not the second.

Reply:
Well it can, since we break if one of them fails.

Reply:
Okay, I did it and it is still working, so :+1: